es6-features-practise/test/iterators.js

```diff
@@ -122,23 +132,23 @@ describe('A simple iterable without items inside, implementing the right protoco
   });
   describe('using the iterable', function() {
     it('it contains no values', function() {
-      let values;
+      let values='';
       for (let value of iterable) {
         values += value;
       }
       assert.equal(values, '');
     });
     it('has no `.length` property', function() {
-      const hasLengthProperty = iterable;
+      const hasLengthProperty =  'length' in iterable ;
```

Comment:
I dont understand this one.

Reply:
You need to check whether length property exists.

you can `iterable.hasOwnProperty('length')
or simply `typeof iterable.length === 'number' `